src/content/blog/astro-520.mdx

```diff
@@ -0,0 +1,183 @@
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Tailwind 4 support
+
+With their release of v4, Tailwind CSS now offers a `@tailwindcss/vite` plugin that can be directly added to your Astro project. This simplifies the Tailwind experience in Astro, and is now the recommended way to use Tailwind 4 in Astro.
+
+Astro 5.2 adds native support for this Vite plugin, and the `astro add tailwind` command will now add the plugin to your Astro config and create a default CSS file that imports Tailwind styles.
+
+As a result, the `@astrojs/tailwind` integration is now deprecated, but will continue to work for older versions of Tailwind. To upgrade to Tailwind 4, please uninstall the integration and either use the updated `astro add tailwind` command or follow the [Tailwind documentation for manual installation](https://tailwindcss.com/docs/installation/framework-guides/astro).
+
+Thanks to Eveeifyeve ([`@Eveeifyeve`](https://github.com/Eveeifyeve)) who did the first work on supporting Tailwind 4 while it was in alpha, even though this ended up using a different implementation.
+
+## Trailing slash redirects
+
+Whether or not a path should have a trailing slash is a common debate in web development. However everyone agrees that it should be consistent. Duplicate content is bad for SEO and confusing for users. Astro has long allowed you to configure whether routes should be matched with or without trailing slash, but previously this has meant returning a "not found" page for the incorrect path or leaving it up to the host to handle conflicts. Now, Astro will automatically redirect to the correct path on routes that are rendered on-demand, if you have configured the `trailingSlash` option in your `astro.config.mjs`.
+
+Now, it doesn't matter whether your visitor navigates to `/about/`, `/about`, or even `/about///`. In production, they'll always end up on the correct page. For GET requests, the redirect will be a 301 (permanent) redirect, and for all other request methods, it will be a 308 (permanent, and preserve the request method) redirect.
+
+During development, Astro will display an error page instead of redirecting to help you catch any misconfigurations and bad links.
+
+To enable trailing slash redirects, set the `trailingSlash` option in your `astro.config.mjs`:
+
+```js title="astro.config.mjs" ins={3}
+export default defineConfig({
+	adapter: node({ mode: 'standalone' }),
+	trailingSlash: 'never', // or 'always'
+});
+```
+
+For more information, see the [`trailingSlash` documentation](https://docs.astro.build/en/reference/configuration-reference/#trailingslash).
+
+## External redirects
+
+Astro now supports defining external redirects straight from the configuration. External redirects must start with `https` or `http`:
```

Comment:
```suggestion
Astro now supports defining external redirects to `http` or `https` destinations straight from the configuration. 

Just like with internal redirects available since Astro 2.9, with an adapter you can also provide an object as the value. This allows you to specify a `status` code in addition to the new `destination`.
```
Tried to beef this up here, and make the connection with existing internal redirects!